test/e2e/specs/specs-playwright/wp-woocommerce__landing-page.ts

```diff
@@ -0,0 +1,63 @@
+import {
+	DataHelper,
+	SidebarComponent,
+	TestAccount,
+	WoocommerceLandingPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'WooCommerce Landing Page' ), function () {
+	let page: Page;
+
+	describe( 'Learn more button opens a support article', function () {
+		let landingPage: WoocommerceLandingPage;
+
+		beforeAll( async () => {
+			page = await browser.newPage();
+
+			const testAccount = new TestAccount( 'defaultUser' );
+			await testAccount.authenticate( page );
+		} );
+
+		afterAll( async () => {
+			await page.close();
+		} );
+
+		it( 'Navigate to WooCommerce (landing page)', async function () {
+			const sidebarComponent = new SidebarComponent( page );
+			await sidebarComponent.navigate( 'WooCommerce' );
+		} );
```

Comment:
Are you all hoping to specifically cover navigation from the sidebar menu item?

If you have that navigation covered elsewhere, we've been slowly moving tests over to navigate directly to the page under test by URL. This helps avoid some potential flakiness from the sidebar and tends to make tests a little faster.

The way this is usually done is by exposing a method called `visit` on the POM class. For example, [see the AccountSettings POM class](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/lib/pages/me/account-settings-page.ts#L44).

That said, if you don't have a test case for navigation covered elsewhere, it seems like it might be worth to keep the sidebar navigation as part of this test (as I know it's new for that menu item to be there for simple site users, and we probably want to make sure that's working okay). I just wanted to make sure to mention the alternative!

Reply:
> Are you all hoping to specifically cover navigation from the sidebar menu item?

In these landing page tests, yes. We have conditional logic on that menu item. (not to mention an [open bug](https://github.com/Automattic/wp-calypso/issues/60429) :smile: )

> The way this is usually done is by exposing a method called visit on the POM class. For example, see the AccountSettings POM class.

Thanks for the tip, will use this method for the singup flow tests as we won't need to re-test the navigation/landing page in those cases.